doc/source/whatsnew/v0.18.0.txt

```diff
@@ -340,4 +340,8 @@ Bug Fixes
 - Bug in ``Index`` prevents copying name of passed ``Index``, when a new name is not provided (:issue:`11193`)
 
 - Bug in ``read_excel`` failing to read any non-empty sheets when empty sheets exist and ``sheetname=None`` (:issue:`11711`)
+
+- Bug in ``read_excel`` failing to raise NotImplemented error when `parse_dates` and `date_parser` are provided (:issue:`11544`)
```

Comment:
use double backticks are ``NotImplementedError`` and say: keywords ``parse_dates`` and ``date_parser``